src/components/App.jsx

```diff
@@ -35,13 +35,9 @@ class App extends Component {
     this._getGISData(this.props.match);
   }
 
-  _getCouncilDistricts = () => {
-    return axios.get('http://data-greensboro.opendata.arcgis.com/datasets/829c58aaaf0c4bf0b59f93bfe3cb4c13_3.geojson');
-  }
+  _getCouncilDistricts = () => { axios.get('http://data-greensboro.opendata.arcgis.com/datasets/829c58aaaf0c4bf0b59f93bfe3cb4c13_3.geojson'); }
 
-  _getCommissionerDistricts = () => {
-    return axios.get('http://data-greensboro.opendata.arcgis.com/datasets/1b60f15bb4dc4d8f96bd4831a8fbf063_5.geojson');
-  }
+  _getCommissionerDistricts = () => { axios.get('http://data-greensboro.opendata.arcgis.com/datasets/1b60f15bb4dc4d8f96bd4831a8fbf063_5.geojson'); }
```

Comment:
Hey @robatronPrime - Thanks so much for your contribution. I worry that this pattern will not implicitly return the axios promise- try instead this pattern:

```
_getCommissionerDistricts = () => axios.get('http://data-greensboro.opendata.arcgis.com/datasets/1b60f15bb4dc4d8f96bd4831a8fbf063_5.geojson');
```

Note the removal of `{ }` so the arrow function will implicitly return a promise.

Sorry for such a delayed response on this! Thanks for your help